restore/README.md

```diff
@@ -0,0 +1,35 @@
+# Restore Instructions
+
+Please follow these instructions if you are restoring any settings that has been missed or deleted during/after the installation.
+
+# Please use this set of instructions:
+
+a) You have a 4.0.3.1 installed and required to restore any settings.
+b) Already have a gitops-repo configured for ISD (AP and Spinnaker) Configuration.
+
```

Comment:
Add a heading saying Restore has two 2 methods.
1. Namespace deleted from the cluster.
2. Resource deleted from the Namespace.
And the single line statements about each one